modules/installation-dns-user-infra.adoc

```diff
@@ -30,6 +30,9 @@ external to the cluster and from all the nodes within the cluster.
 for the control plane machines. This record must be resolvable from all the
 nodes within the cluster.
 
+|
```

Comment:
@brenton, @soltysh, is there specific guidance that we can provide in this DNS table to address the domain/host issue in the bug?

Reply:
I'll defer to @ironcladlou since he owns that area.

Reply:
I'd probably add a note here that explains the importance of the API Server being able to resolve the worker nodes by the hostname recorded in Kubernetes.  You could mention that subtle problems with proxied API calls can happen otherwise.  You could show how using 'oc' to fetch logs from a pod would exhibit the problem.